src/containers/ServiceWorker/ServiceWorker.tsx

```diff
@@ -0,0 +1,30 @@
+import useNotifier from "@saleor/hooks/useNotifier";
+import React from "react";
+import { useIntl } from "react-intl";
+import { useServiceWorker } from "src/hooks/useServiceWorker";
+
+import messages from "./messages";
+
+const ServiceWorker: React.FC = () => {
+  const { updateAvailable } = useServiceWorker(60 * 1000);
+  const notify = useNotifier();
+  const { formatMessage } = useIntl();
+
+  React.useEffect(() => {
+    if (updateAvailable) {
+      notify({
```

Comment:
If user dismissed the dialog we should stop showing it to him every minute and unsubscribe from the listener.